### PR TITLE
Updating doc/table*md v2 -> v3

### PR DESCRIPTION
**Pull request by a27cheung**
Updating version

### CODE REVIEW

**Comment by a27cheung** [comment]
Resolved merges w/intro paragraph

File: docs/table-epidemiology.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ Information related to the COVID-19 infections for each date-region pair.
 
 ## URL
 This table can be found at the following URLs depending on the choice of format:
-* [epidemiology.csv](https://storage.googleapis.com/covid19-open-data/v2/epidemiology.csv)
-* [epidemiology.json](https://storage.googleapis.com/covid19-open-data/v2/epidemiology.json)
+* [epidemiology.csv](https://storage.googleapis.com/covid19-open-data/v3/epidemiology.csv)
+* [epidemiology.json](https://storage.googleapis.com/covid19-open-data/v3/epidemiology.json)
 
 
 ## Schema
```
Comment (owahltinez) [inline docs/table-epidemiology.md:13]: Missing updates to the schema which were part of the v2 -> v3 migration (e.g. `total_` -> `cumulative_`). Other tables might be affected too, like hospitalizations and vaccinations.

**Review by owahltinez — CHANGES_REQUESTED** [review]

File: docs/table-epidemiology.md (hunk 1 above)
Comment (owahltinez) [reply docs/table-epidemiology.md:13]: Thanks @a27cheung! It looks like you have only updated the schema docs for the epidemiology table, there are several others that need updating as well — the details are in the migration doc that you wrote a few months ago.

**Comment by a27cheung** [comment]
Updated total_ -> cumulative

File: docs/table-epidemiology.md (hunk 1 above)
Comment (a27cheung) [reply docs/table-epidemiology.md:13]: Np! Haven't forgotten; just prefer many small changes. :)

**Review by a27cheung — COMMENTED** [review]
Changed total_ -> cumulative_

**Review by a27cheung — COMMENTED** [review]
Total -> cumulative

**Comment by a27cheung** [comment]
Merged all conflicts and finished up v2 to v3 here according to reference doc.

**Comment by owahltinez** [comment]
We're still missing some of the changes — for example renaming of some hospitalization fields. Can you please verify using the actual output as reference, not the documentation which is apparently incomplete?

**Comment by owahltinez** [comment]
There's also conflicts which prevent merging, please run `git rebase main`, fix the conflicts locally, and then run `git push -f` to upload your changes

**Comment by owahltinez** [comment]
Closing in favor of #490

**Pull request closed without merging**